lib/master.js

```diff
@@ -8,6 +8,7 @@ const BaseService = require('./base_service');
 const Worker = require('./worker');
 const Logger = require('./logger');
 const RateLimiterMaster = require('./ratelimiter').master;
+const MetricsEndpoint = require('./prometheus_server');
```

Comment:
Same problem as above. Please create a generic class that is able to handle all types of metrics.

Reply:
This is an HTTP server that exposes the Prometheus metrics endpoint.  I'm not sure what you're asking for here.  Do you think the HTTP server implementation should be in a different file?